das/dasRpcClient.go

```diff
@@ -83,6 +89,11 @@ func (c *DASRPCClient) Store(ctx context.Context, message []byte, timeout uint64
 		rpcClientStoreDurationHistogram.Update(time.Since(start).Nanoseconds())
 	}()
 
+	if c.disableChunkedStore {
+		log.Info("Legacy store is being force-used by the DAS client", "url", c.url)
```

Comment:
Does it make sense to set this log as info? I fear that this will flood the logs.

Reply:
I will change this to debug